dot/parachain/network-bridge/messages/pov_fetching.go

```diff
@@ -26,6 +27,14 @@ func (p *PoVFetchingRequest) Decode(in []byte) (err error) {
 	return scale.Unmarshal(in, p)
 }
 
+func (p *PoVFetchingRequest) Response() network.ResponseMessage {
+	return &PoVFetchingResponse{}
+}
```

Comment:
Do we need this method?
What is the use of this method? Can you explain?

Reply:
The method is part of the [interface `ReqProtocolMessage`](https://github.com/ChainSafe/gossamer/blob/b90f91c038a2dbc607a7c6174f192b434e84fe19/dot/parachain/network-bridge/messages/request_response_protocols.go#L48). The code that calls it does not know the type to use for decoding the response.